### PR TITLE
Set up Prettier code style

### LINKED ISSUE #68: Set up Prettier to maintain code style consistency
Following discussion in https://github.com/simple-icons/simple-icons-font/pull/66#discussion_r551003162, add also Javascript files configuration for `.editorconfig` file.

### PR DESCRIPTION
**Pull request by mondeja**
This is the default code style that Prettier imposes. Except the examples in "Usage" section of the README, I'm seeing everything fine, but before continuing I would like to see your feedback @ericcornelissen. Do you want to enforce style in Markdown files? And other types of files that are not Javascript?

I've added a job to run Prettier, ~~named "Lint", but I'm not sure if is the correct term... :thinking:~~ ~~Definitively it is confusing, so I've changed by "Enforce code style" in job name and "prettier" in NPM script name.~~ EDIT: Changed by sugerence.

Closes #68

### CODE REVIEW

**Comment by ericcornelissen** [comment]
Hmm, here is my initial reaction (not a full review):

- We should definitely just ignore `LICENSE.md` (as that file simply never has to be touched nor formatted).
- I would be inclined to agree that we should probably ignore all `.md` files. I really dislike the changes to the examples it is producing in the README.md (particularly the **Usage** examples). That said, we might be able to configure Prettier so as to still have some of its benefits in `.md` files while avoiding thesee changes... I can look into this later.
- **If** we do decide to add configuration for prettier, I would be inclined to configure it to use single quotes rather than double quotes, as that seems to be the standard we're using in the organisation at large. If that is the only thing we want to configure, I don't feel as strongly about it
- I would like to see all `const`, `let`(, `var`?) statements with multiple values `require`d transformed into one `const`-or-`let`-per-`require`. But we may leave that to a separate PR.

> I've added a job to run Prettier, named "Lint", but I'm not sure if is the correct term...

As far as I know that is the correct term :+1: Could you also add an NPM command name `format` that runs prettier and formats the code of the repository?

**Comment by mondeja** [comment]
> We should definitely just ignore LICENSE.md (as that file simply never has to be touched nor formatted).
I would be inclined to agree that we should probably ignore all .md files.

Ignored all Markdown files because the only "improvement" was to enforce list item characters to be `-`, but to maintain the consistency with main repository we should keep `*`.

> If we do decide to add configuration for prettier, I would be inclined to configure it to use single quotes rather than double quotes...

Configured Prettier with single quotes without configuration file: since it's the only configuration that we need (at least for now) I've used `--single-quote` argument in scripts.

> I would like to see all const, let(, var?) statements with multiple values required transformed into one const-or-let-per-require. But we may leave that to a separate PR.

Done in this :+1:

File: scripts/build.js
Comment (ericcornelissen) [inline scripts/build.js:11]: I feel like you misunderstood my comment

> I would like to see all const, let(, var?) statements with multiple values required transformed into one const-or-let-per-require. But we may leave that to a separate PR.

I actually prefer the old style (see [this comment]() for why). Do you have any particular reason for changing it into this?

**Review by ericcornelissen — CHANGES_REQUESTED** [review]

**Comment by ericcornelissen** [comment]
One more thing, it seems Prettier is not acting on [the `.pug` file](https://github.com/simple-icons/simple-icons-font/blob/c94dafa63c0cedd4dc6de2e74b99b89c96a2d7f4/preview/html/testpage.pug) while I think it should. Also, the `.pug` extension should probably be included in the `.editorconfig` file then.

**Comment by mondeja** [comment]
I've included the changes that you've proposed, configuring pug format lint using [`prettier/plugin-pug`](https://github.com/prettier/plugin-pug).

File: .editorconfig
Comment (ericcornelissen) [inline .editorconfig:13]: This looks good to me now. Before we merge this, should we just set this configuration for `[*]`? If not, I suggest we change this to:

```suggestion
[*.{css,js,json,pug,yml}]
indent_size=2
indent_style=space
```

Comment (mondeja) [reply .editorconfig]: > Before we merge this, should we just set this configuration for `[*]`?

Yes :+1:

**Comment by ericcornelissen** [comment]
Awesome, thanks for you efforts on this one @mondeja  :tada:

**Pull request merged**
